### PR TITLE
IAM beta 2 doc updates

### PR DESCRIPTION
**Pull request by rymorale**
## Description
Update docs for IAM beta 2 based on the iam inbound changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Documentation/sources/api-usage.md [hunk 1]
```diff
@@ -49,15 +61,27 @@ Messaging.handleNotificationResponse(intent, true, null);
 ```
 
 ##### Sending feedback when application is opened with custom action. 
-To do this, add the following code where you have access to `intent` after the user interact with the push notification:
+Call the `handleNotificationResponse` API with a custom action in the
```
Comment (rymorale) [inline Documentation/sources/api-usage.md:64]: just noticed the hanging `the`... i'll fix this

File: Documentation/sources/how-to-messaging-delegate.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+	 */
+	void onShowFailure();
+}
+```
+
+### Retrieving the Message object from the implemented interface methods
+
+The user interface methods (except for `onShowFailure()`) in a `FullscreenMessageDelegate` implementation will be passed an `AEPMessage` object. An `AEPMessage` object is the Android Core implementation of the `FullscreenMessage` interface. It contains a reference to the parent `Message` class and is your primary way to interact with the message.
+
+A reference to the `AEPMessage` object can be obtained by calling `fullscreenMessage.getParent()` . An example of how to access the `Message` in the `onShow` delegate method can be seen below:
```
Comment (sbenedicadb) [inline Documentation/sources/how-to-messaging-delegate.md:72]: are all these references to `AEPMessage` correct?

File: Documentation/sources/how-to-messaging-delegate.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    
+    // cast to MessageWebView to access the startInAppMessage function
+    inAppMessageView = (MessageWebView) currentMessage.getWebView();
+    
+    return false;
+  }
+  
+  return true;
+}
+```
```
Comment (sbenedicadb) [inline Documentation/sources/how-to-messaging-delegate.md:147]: i'm missing context from this example - unsure what `MessageWebView` is

**Review by sbenedicadb — CHANGES_REQUESTED** [review]
caught a couple small issues

File: Documentation/sources/how-to-messaging-delegate.md (hunk 2 above)
Comment (rymorale) [reply Documentation/sources/how-to-messaging-delegate.md:72]: good call out, i had to clean this up

File: Documentation/sources/how-to-messaging-delegate.md (hunk 3 above)
Comment (rymorale) [reply Documentation/sources/how-to-messaging-delegate.md:147]: A `MessageWebView` object is an extension of an android `WebView`. I should add the call to startInAppMessage for more context.

**Review by sbenedicadb — APPROVED** [review]

**Pull request merged**